### ISSUE TITLE
The autocompletion popup is all jumpy because of the different size of the snippets

### ISSUE THREAD

**Issue by rgehan** [onivim/oni#1758]
When moving in the list of the suggested snippets, the preview area is updated with the currently focused snippet. The problem is that the snippets are not the same size, making the list jump around.

When presenting a list to the user, the list should move as little as possible so that is is easy to target a specific item.

![capture d ecran 2018-03-09 a 09 50 01](https://user-images.githubusercontent.com/793402/37198634-424a5d98-237f-11e8-9fe9-3bd8cfdc6a07.png)
![capture d ecran 2018-03-09 a 09 49 52](https://user-images.githubusercontent.com/793402/37198635-4265d41a-237f-11e8-944b-6ae94b620c4f.png)

See below; the preview areas are not the same size, and since the popup is fixed from the bottom, it makes the list move up & down.

**Comment by bryphe** [comment]
Yes, good call - the list should never move while the user is navigating. I wonder if we should pivot and have the preview area to the right of the completion menu (or left, if near the right edge of the screen). I believe this is how some other completion experiences handle it.
